### PR TITLE
Route from discussion list to discussion detail component

### PR DESCRIPTION
**Pull request by Linzeur**
- Fix routes in index component
- Add Link to redirect to thread component in discussion list component

### CODE REVIEW

File: src/views/thread.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ function Thread({ discussionId }) {
 
   const discussionList = JSON.parse(localStorage.getItem("discussionList"));
 
-  const discussion = discussionList.find(e => e.id === +discussionId);
+  const discussion = discussionList.find(e => e.id === +id);
```
Comment (DiegoCuevas) [inline src/views/thread.js]: Could you explain me this line?

Comment (diegotc86) [reply src/views/thread.js]: Find method iterates through the array and return the first element which satisfies the condition given. In this case "e" receive the value of each element of discussionList (a discussion Object) and compare the key "id" of the element with the "id" received via props.
TL;DR: We are finding the discussion object on the discussionList who has the same id of "id" received via props.
More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Comment (Linzeur) [reply src/views/thread.js]: This array returns the index where it finds the conditional is true. We use that to get the discussion object from our discussion array
